portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -21,28 +21,39 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
+import java.util.Arrays;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-
-    // Define locations that Fatima has lived in 
-    ArrayList<String> fatimaLocations = new ArrayList<String>();
-    fatimaLocations.add("Lansing, Michigan");
-    fatimaLocations.add("Scotts Valley, California");
-    fatimaLocations.add("Cupertino, California");
-    fatimaLocations.add("Irvine, California");
-
-    // Convert the locations to JSON
-    String json = convertToJsonUsingGson(fatimaLocations);
-   
-    // Send the JSON as the response
-    response.setContentType("application/json;");
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    ArrayList<String> comments = new ArrayList<String>();
+    String userComments = getParameter(request, "text-input", "");
+    String[] words = userComments.split("\\s*,\\s*");
+    
+    for (String comment : words) {
+        comments.add(comment);
+    }
+    
+    // Respond with the result.
+    response.setContentType("application/json");
+    String json = new Gson().toJson(comments);
     response.getWriter().println(json);
-
+  }
+  
+  /**
+   * @return the request parameter, or the default value if the parameter
+   *         was not specified by the client
```

Comment:
can you also add a line about what the function does and add @param for each parameters? you can find javadoc examples online - i can point you to some if you'd like

Reply:
Resolved in PR #9